pkg/gds/client/profiles_test.go

```diff
@@ -0,0 +1,196 @@
+package client_test
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+
+	"github.com/shibukawa/configdir"
+	"github.com/stretchr/testify/require"
+	"github.com/trisacrypto/directory/pkg/gds/client"
+	"gopkg.in/yaml.v2"
+)
+
+const profileYAML = "profiles.yaml"
```

Comment:
Perhaps we should capitalize the constants [`profileYaml` and `profileVersion`](https://github.com/trisacrypto/directory/blob/main/pkg/gds/client/profiles.go#L39-L42) so we can import them and use them in the tests? Or we could create a helper function `func GetProfileMeta() (yaml, version string)` that returns them without being able to modify them if you'd prefer.